src/infrastructure/storage/page.ts

```diff
@@ -0,0 +1,36 @@
+import type { Page } from '@/domain/entities/Page';
+import { SubscribableStore } from './abstract/subscribable';
+
+export type TabStoreData = {
+  /**
+   * Array of tabs to be stored
+   */
+  openedPages: Page[];
+};
+
+export class PageStore extends SubscribableStore<TabStoreData> {
```

Comment:
>PageStore

name is not clear. Maybe WorkspaceStore?

Reply:
i guess we store opened pages (renamed to `OpenedPagesStore`), not workspaces, same as users and notes